src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -569,8 +569,10 @@ def load_arguments(self, _):
             for dest in scaleset_name_aliases:
                 c.argument(dest, vmss_name_type, id_part=None)  # due to instance-ids parameter
 
-    with self.argument_context('vmss create') as c:
+    with self.argument_context('vmss create', operation_group='virtual_machine_scale_sets') as c:
```

Comment:
What is the impact of `operation_group='virtual_machine_scale_sets'`?

Reply:
To make sure the following api version take effect
https://github.com/Azure/azure-cli/blob/a8dce7f77b62d07b28f3e969d24ffe4c96ad5a8a/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L149